components/MashCards/FinalDecisionCard.spec.tsx

```diff
@@ -25,4 +26,29 @@ describe('FinalDecisionCard', () => {
     expect(screen.getByText('Referral category'));
     expect(screen.getByText('Screening decision'));
   });
+  it('renders the high priority banner when the referral is marked as urgent in the previous stage', () => {
```

Comment:
Similar comment to the one added for `ScreeningCard` component but this has a clearer test description, nice! 👍